client/src/pages/Home.js

```diff
@@ -309,7 +310,13 @@ const SavedSearches = props => {
   let savedSearches = []
   if (data) {
     savedSearches = data.savedSearches
-    if (savedSearches && savedSearches.length > 0 && !selectedSearch) {
+    if (_isEmpty(savedSearches)) {
```

Comment:
No need for nested if-statements (and-condition will satistfy)

Reply:
Nope, the else needs to be done if and only if `savedSearches` exists and is not empty…